alacritty/src/event.rs

```diff
@@ -662,8 +663,28 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
 
     /// Process a new character for keyboard hints.
     fn hint_input(&mut self, c: char) {
-        self.display.hint_state.keyboard_input(self.terminal, c);
+        let action = self.display.hint_state.keyboard_input(self.terminal, c);
         *self.dirty = true;
+
+        let HintMatch { text, action } = match action {
+            Some(action) => action,
+            None => return,
+        };
+
+        match action {
+            // Launch an external program.
+            HintAction::Command(command) => {
+                let mut args = command.args().to_vec();
+                args.push(text);
+                start_daemon(command.program(), &args);
+            },
+            // Copy the text to the clipboard.
+            HintAction::Action(HintInternalAction::Copy) => {
+                self.clipboard.store(ClipboardType::Clipboard, text);
+            },
+            // Write the text to the PTY.
+            HintAction::Action(HintInternalAction::Paste) => self.write_to_pty(text.into_bytes()),
```

Comment:
Given that you may be inside a search while hints are active, I might expect the text to go to it's buffer rather than the PTY. This seems potentially complex, but worth considering. Having text pasted to a potentially invisible cursor seems undesirable. Vi-mode might be an exception, however I might just request that the `Paste` action is a no-op in that case perhaps.

Reply:
It's been a while, but I'm pretty sure I've considered that and wouldn't expect things to behave this way. Using hints to paste into the selection regex just seems somewhat odd to me.

Reply:
Decided to just reuse the existing `paste` function from `input.rs`. This way it's at least uniform everywhere.